packages/apps/graph/src/graph/subscription/events.ts

```diff
@@ -121,6 +134,12 @@ async function getLastEvents(
           column: 'parameters',
         }),
       }),
+      ...(minimumDepth && {
+        OR: await getConditionForMinimumDepth(
+          minimumDepth,
+          chainId ? [chainId] : chainIds,
```

Comment:
I wouldn't pass the default chainIds here since it makes the query more complex. See other comment.

```suggestion
          chainId ? [chainId] : undefined,
```